config/payment.php

```diff
@@ -120,6 +120,14 @@
             'pubKey' => '',
             'currency' => 'T', //Can be R, T (Rial, Toman)
         ],
+        'jibit' => [
+            'apiPaymentUrl' => 'https://napi.jibit.ir/ppg/v3',
+            'apiKey' => '',
+            'secretKey' => '',
+            'tokenStoragePath' => function_exists('storage_path') ? storage_path('jibit/') : 'jibit/',
```

Comment:
nice one =) good job.
```suggestion
           // you can change the token storage path in Laravel like this 
           // 'tokenStoragePath' => function_exists('storage_path') ? storage_path('jibit/') : 'jibit/'
           'tokenStoragePath' => 'jibit/',
```

please remove `storage_path` and instead add a comment instead. since this package is not specificly for Laravel, we should not couple it with Laravel functions.